robot/camera.py

```diff
@@ -76,7 +73,7 @@ def _see_to_results(data):
         for token in data["markers"]:
             markers.append(Marker(token))
         # Sort by distance
-        return ResultList(sorted(markers, key=lambda x: x.distance_metres))
+        return ResultList(markers)
```

Comment:
I would still argue this list should be sorted by distance. Unless it's handled elsewhere?

Reply:
without support of Polar co-ordinates it's not trivial to sort by distance any more. Feature coming once https://github.com/sourcebots/sb-vision/pull/5 is merged